lib/pbench/test/unit/server/query_apis/test_datasets_contents.py

```diff
@@ -142,14 +145,16 @@ def test_query(
             auth_json, build_auth_header["header_param"]
         )
 
-        response = query_api_get(
+        response = query_api(
             self.pbench_endpoint,
             self.elastic_endpoint,
+            self.payload,
```

Comment:
`self.payload` is always `None`, here, right?  If that's the case, I recommend changing this line to
```
           payload=None,
```
Ditto line 263, et al.